src/ui/link/index.tsx

```diff
@@ -103,6 +104,7 @@ const Link = forwardRef<HTMLButtonElement | HTMLAnchorElement, LinkProps>(
     const handleLinkPress = (e: MouseEvent | KeyboardEvent) =>
```

Comment:
nit: Given this is immediately split into keyboard vs non-keyboard inside `handleNavigation`, might as well just fully split this into two different things